### ISSUE TITLE
Feature Request: Add option for ASC/DESC 

### ISSUE THREAD

**Issue by SippieCup** [jonschlinkert/array-sort#5]
Easy enough, but just adding a third function argument for sort direction.

**Comment by doowb** [comment]
You can use the `reverse` option. This is the same as sorting in descending order. See the [usage example](https://github.com/jonschlinkert/array-sort#usage) to see if this is what you're looking for.

**Comment by doowb** [comment]
Closing this... you can use `reverse` to get the array sorted in descending order.

**Comment by SippieCup** [comment]
Sorry, Missed this before. didn't realize it was `reverse` until you told me and guess I never responded and thanked you. 

too used to SQL... 

Thanks.

**Comment by mesqueeb** [comment]
I hope you can change the syntax to `direction: 'asc'` and `direction: 'desc'`, because lots of people are more used to this IMO!

**Comment by jonschlinkert** [comment]
I agree with @doowb but also see the value of having `asc`/`desc`. I think we should consider adding those.

**Comment by doowb** [comment]
I'll try to work on this sometime along with #9. I think there are concepts that we can look at to make this easier to do.

**Comment by tywei90** [comment]
You can try this [arr-sort](https://github.com/tywei90/arr-sort). 

Sort an object array by one or more properties even nested properties. Besides, you can determine the direction even supply a comparison function in each property sorting.
